kvmagent/kvmagent/plugins/doc/port_mirror_to_physical_nic.md

```
@@ -0,0 +1,259 @@
+  - `brctl show`
+- 日志：
+  - KVM Agent 日志中有应用/释放成功与调试信息（`PortMirrorPlugin`）。
+
+
+## 镜像到物理机接口
+前述功能可以把流量镜像到虚拟机vnic, 本次改进将流量镜像到物理机网卡，或者物理机网卡的vlan子接口
+
+### 如何判断镜像目的
+参数mirror新增两个参数: 
+- `interfaceName`: 物理机网卡的命令
+- `vlanId `: vlan子接口的vlan id, 0 表示不使用vlan子接口
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)